earth_enterprise/src/fusion/autoingest/plugins.src/RasterProject.src

```diff
@@ -1085,15 +1097,20 @@ uint32 RasterProjectAssetVersionImplD::BuildIndex(
 
     notify(NFY_PROGRESS, "BuildIndex finding intersections for inset %d of %lu",
            i+1, genInfo.size());
-
+    int maxQuadTreeLevel = 0;
+    QuadtreePath coverageMBR = localIndex.getQuadtreeMBR(genInfo[i].coverage.degreeExtents(), maxQuadTreeLevel, endMinifyLevel);
+    std::vector<const InsetInfo<RasterProductAssetVersion>*> qualifiedInsetInfoPtrs = localIndex.intersectingExtents(coverageMBR, genInfo[i].coverage.beginLevel(), endMinifyLevel);
```

Comment:
Just checking - was the removal of the call to `PreprocessForInset` here intentional? I think it's fine, I just want to make sure it's what was intended.

Reply:
Yeah, that was on purpose. I initially called `PreprocessForInset` after `intersectingExtents`, then I went a step further to take out the call to `PreprocessForInset`.
My thinking was that since we're theoretically just trimming out things we know we won't need anyway, `intersectingExtents` might be effective enough on its own. That seems to be the case so far.

Reply:
Makes sense.